src/pages/admin/doi.js

```diff
@@ -40,6 +41,20 @@ export default function Doi() {
         [router]
     );
 
+    const onRouteMetadataView = useCallback(
+        (path, type, resourceId) => {
+            router.push({
+                pathname: `/${NavigationConstants.DATA}/${constants.DATA_STORE_STORAGE_ID}${path}`,
+                query: {
+                    type,
+                    resourceId,
+                    view: NavigationParams.VIEW.METADATA,
```

Comment:
Only the `view` query param should be required for viewing metadata, so the `type` and `resourceId` could be removed from this callback.